### ISSUE TITLE
setup flow

### ISSUE THREAD

**Issue by Andykmcc** [Andykmcc/project-graph#4]
investigate using Flow. try it out, see if it is worth while.

https://flowtype.org/

**Comment by vincepmartin** [comment]
I agree with you about using Flow Comments if we are going to actually use Flow. It would be a huge pain to pull it out later.
